packages/block-directory/src/components/downloadable-block-list-item/index.js

```diff
@@ -92,23 +95,27 @@ function DownloadableBlockListItem( { composite, item, onClick } ) {
 	return (
 		<CompositeItem
 			__experimentalIsFocusable
```

Comment:
I believe we should be able to pass the `__experimentalIsFocusable` prop directly to `Button` too?